PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -11,62 +11,104 @@ enum SearchScope: String, CaseIterable {
 }
 
 class SearchViewModel: ObservableObject {
-
+    static let recentSearchesKey = "SearchViewModel.recentSearches"
```

Comment:
Just a thought - maybe we should prefix keys with the _function / action_ they're related to rather than the type name. For example, `Search.recentSearches`, since the key is utilized with search in general.

Reply:
will update!

Reply:
updated! I agree with this as a general practice going forward. I added it to our style guide as well. Thanks!
https://github.com/Pocket/pocket-ios/wiki/Pocket-Swift-Style-Guide#naming-keys